symfony/framework-bundle/4.2/public/index.php

```diff
@@ -12,7 +12,7 @@
     if (!class_exists(Dotenv::class)) {
         throw new \RuntimeException('APP_ENV environment variable is not defined. You need to define environment variables for configuration or add "symfony/dotenv" as a Composer dependency to load variables from a .env file.');
     }
-    (new Dotenv())->load(__DIR__.'/../.env');
+    (new Dotenv())->loadForEnv('dev', __DIR__.'/../.env');
```

Comment:
The first parameter should not be dynamic ?

Reply:
not by default

Reply:
I understand but some people still use the Dotenv for production and it could lead to errors